webpack/ForemanTasks/Components/TasksTable/Components/SelectAllAlert.js

```diff
@@ -17,14 +17,13 @@ export const SelectAllAlert = ({
         Math.min(itemCount, perPage)
       )}
       <Button bsStyle="link" onClick={selectAllRows}>
-        {__('Select All')}
-        <b> {itemCount} </b> {__('tasks.')}
+        {sprintf(__('Select all %s tasks'), `<b>${itemCount}</b>`)}
```

Comment:
This is an example where proper pluralization should be used, but at least now there is a single sentence rather than a collection of words that may be moved around in other languages.

Reply:
Looks like `<b>` is getting escaped. How would I write this in a compatible way?